test/NuGet.Clients.Tests/NuGet.CommandLine.Test/NuGetRestoreCommandTest.cs

```diff
@@ -2220,10 +2220,88 @@ var targetFileContents
                     waitForExit: true);
 
                 // Assert
-                Assert.True(0 == r.Item1, r.Item2 + " " + r.Item3);               
+                Assert.True(0 == r.Item1, r.Item2 + " " + r.Item3);
                 var packageFileA = Path.Combine(randomSolutionFolder, "GlobalPackages", "packagea","1.1.0", "packageA.1.1.0.nupkg");
                 Assert.True(File.Exists(packageFileA));
             }
         }
+
+        [Fact]
+        public void RestoreCommand_WithAuthorSignedPackage_Succeeds()
```

Comment:
Wouldn't this test run on Mono?

Reply:
Yes.  The CI test run shows that this test passed on Mono.

The expectation at this point is that signed package validation will fail but the package will still restore as an unsigned package, at least until https://github.com/mono/mono/issues/7456 is fixed.